apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -159,7 +160,7 @@ public function enqueue_assets() {
 	 */
 	public function fetch_vertical_data() {
 		$vertical_id        = get_option( 'site_vertical', 'default' );
-		$transient_key      = implode( '_', [ 'starter_page_templates', $vertical_id, get_locale() ] );
+		$transient_key      = implode( '_', [ 'starter_page_templates', 'v2', $vertical_id, get_locale() ] );
```

Comment:
added `v2` as our API shape changed in a breaking way and we don't want the old one to get used from the cache

Reply:
I think it would be fine to omit that, cache invalidates after 24h